src/util/triplea/mapXmlCreator/MapXmlCreator.java

```diff
@@ -1073,7 +1072,6 @@ static void saveXML() {
       Logger.getLogger(MapXmlCreator.MAP_XML_CREATOR_LOGGER_NAME).log(Level.INFO,
           "Game XML written to " + newFile.getCanonicalPath());
     } catch (final IOException | HeadlessException | TransformerException | ParserConfigurationException e) {
-      // TODO Auto-generated catch block
```

Comment:
Same, error handling should be improved, log the exception, and the value of any interesting method parameters.